cli/mngcheckpoint/mngcheckpoint.go

```diff
@@ -0,0 +1,136 @@
+package mngcheckpoint
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"strconv"
+
+	"github.com/spf13/cobra"
+	"github.com/tarantool/tt/cli/cmdcontext"
+)
+
+// CmdFlags contains flags for managing checkpoint files commands.
+// Used for commands tt cat and tt play, which are checkpoint files commands.
+type CmdFlags struct {
+	To         uint64
+	From       uint64
+	Space      []int
+	Replica    []int
+	ShowSystem bool
+}
+
+// InitFlags sets initial values to flags of managing checkpoint files commands.
+// Used for commands tt cat and tt play, which are checkpoint files commands.
+func (cf *CmdFlags) InitFlags(cmd *cobra.Command) {
+	cmd.Flags().Uint64Var(&cf.To, "to", 18446744073709551615, "Show operations ending with the given lsn")
+	cmd.Flags().Uint64Var(&cf.From, "from", 0, "Show operations starting from the given lsn")
+	cmd.Flags().IntSliceVar(&cf.Space, "space", nil, "Filter the output by space number. May be passed more than once")
+	cmd.Flags().IntSliceVar(&cf.Replica, "replica", nil, "Filter the output by replica id. May be passed more than once")
+	cmd.Flags().BoolVar(&cf.ShowSystem, "show-system", false, "Show the contents of system spaces")
+}
+
+// FillCatEnv fills the internal env vars of cat cmd with cli args.
+func FillCatEnv(args []string, catFlag CmdFlags) error {
+	if len(args) == 0 {
+		return fmt.Errorf("It is required to specify at least one .xlog or .snap file.")
+	}
+
+	// List of files is passed to lua cat scipt via environment variable in json format.
+	filesJson, err := json.Marshal(args)
+	if err != nil {
+		log.Fatalf("Internal error: problem with creating json params with files: %s", err.Error())
```

Comment:
Use `util.InternalError` for such cases (here and bellow).

Reply:
Done